### ISSUE TITLE
Skaled accepted new transactions but not inserted in blocks

### ISSUE THREAD

**Issue by oleksandrSydorenkoJ** [skalenetwork/skaled#1726]
**Note:**
There isn't a problem with the same versions of skaled, the problem is only during the update procedure.

**Versions:**
11 nodes - 3.17.1
5 nodes - 3.16.1

**Preconditions**
Patch timestamps on 3.17.1 don't enabled
Active schain medium type 

**Steps to reproduce**
1. Run load tests on all endpoints
```
1000 users 
token mint\transfer
MTM =15 
at least 15 million block gas usage 
and 125 KB block size 
``` 
2. Wait for a few hours 

**Expected state**
All valid transactions should be mined without any stucks.

**Actual state:** 
Some transactions are locked in the current transactions queue and aren't mined over time.  
After stopping the load they are not broadcasted.

**Comment by PolinaKiporenko** [comment]
Outcome from https://github.com/skalenetwork/skaled/issues/1398

**Comment by oleksandrSydorenkoJ** [comment]
For 3.16.1 Skaled has more capacity in a transaction queue, so right now it is not a problem. 
For 3.17.1 Skaled has improved transaction's validation procedure. 
Should be fixed after updating to  3.17.1 on all of nodes
Workaround if transaction queue fulfilled to 1000 transactions during the update is just send transactions from users with lower Nonce.

**Comment by DmytroNazarenko** [comment]
Duplicate of https://github.com/skalenetwork/skaled/issues/1726
